superset/models/dashboard.py

```diff
@@ -242,6 +242,12 @@ def changed_by_name(self) -> str:
             return ""
         return str(self.changed_by)
 
+    @property
```

Comment:
I think this is the same case as the owners and should be handled by the frontend.

Reply:
@michael-s-molina This case is different. The property 'created_by' does not currently exist, therefore it is being added as a new property. The interface used for this addition is 'changed_by_name', which closely resembles the desired shape.

Reply:
Sounds good